### ISSUE TITLE
An error occurred: Failed to get successful response from server

### ISSUE THREAD

**Issue by TheStoneMX** [danielmonettelli/dotnetmaui-chatgpt-app-oss#5]
Super good job amigo, but I can't get it to work....

when I call var response = await client.PostAsync(APIConstants.OpenAIEndpoint_Completions, content); I get this error, any ideas?

thank you.
~~~
public async Task<string> AskQuestion(string query)
        {
            try
            {
                var completion = new CompletionRequest()
                {
                    Prompt = query
                };

                var body = JsonSerializer.Serialize(completion);
                var content = new StringContent(body, Encoding.UTF8, "application/json");

                var response = await client.PostAsync(APIConstants.OpenAIEndpoint_Completions, content);

                if (response.IsSuccessStatusCode)
                {
                    var data = await response.Content.ReadFromJsonAsync<CompletionResponse>(options);
                    return data?.Choices?.FirstOrDefault().Text;
                }
                else
                {
                    throw new Exception("Failed to get successful response from server");
                }
            }
            catch (Exception)
            {
                // Handle exception or log the error here
                // await Application.Current.MainPage.DisplayAlert("Error", $"An error occurred: {ex.Message}", "OK");
                throw;
            }
        }
~~~

**Comment by TheStoneMX** [comment]
My bet....

I had temperature = 7 ..... O forgot to use 0. to be 0.7 as temperature cant exeed 1
I am just leaving this for anyone else/

**Comment by danielmonettelli** [comment]
Hello @TheStoneMX, apologies for the delayed response. The 403 error you experienced was due to the deprecation of the `text-davinci-003` model. We've since transitioned to the `gpt-3.5-turbo-instruct` model, which I've already updated. Thank you for bringing this issue to our attention.
https://github.com/danielmonettelli/dotnetmaui-chatgpt-oss/commit/586bded463888a0ce77f87bbc4c172d9a0ceddd6
